app/i18n/de/conf.php

```diff
@@ -178,7 +178,7 @@
 			'view' => 'Ansicht',
 		),
 		'hide_read_feeds' => 'Kategorien & Feeds ohne ungelesene Artikel verstecken (funktioniert nicht mit der Einstellung „Alle Artikel zeigen“)',
-		'img_with_lazyload' => 'Verwende die "träges Laden"-Methode zum Laden von Bildern',
+		'img_with_lazyload' => 'Verwende die „träges Laden“-Methode zum Laden von Bildern',
```

Comment:
This one looks weird

Reply:
Let's get @math-GH to tell us what is most usual in German. It is picked from a similar line a bit higher up

Reply:
" is the normal version in German. 

Apple devices uses „“ by default

Reply:
@Alkarex, you are right. `„“` are the typographic correct characters. While this characters are not available on the keyboard we use the `"`, but it is typographically correct a `inch` character.

Reply:
Fixed everywhere